rclcpp/include/rclcpp/context.hpp

```diff
@@ -398,20 +398,22 @@ class Context : public std::enable_shared_from_this<Context>
 
   using ShutdownCallback = ShutdownCallbackHandle::ShutdownCallbackType;
 
+  template<ShutdownType shutdown_type>
   RCLCPP_LOCAL
   ShutdownCallbackHandle
   add_shutdown_callback(
-    ShutdownType shutdown_type,
     ShutdownCallback callback);
 
+  template<ShutdownType shutdown_type>
   RCLCPP_LOCAL
   bool
   remove_shutdown_callback(
-    ShutdownType shutdown_type,
     const ShutdownCallbackHandle & callback_handle);
 
+  template<ShutdownType shutdown_type>
+  RCLCPP_LOCAL
```

Comment:
Not having the `RCLCPP_LOCAL` here seemed like an oversight.

Reply:
good eye 👁️